packages/clients/diplan/src/plugins/Gfi/Content.vue

```diff
@@ -1,13 +1,217 @@
+          <template #default>
+            <tbody>
+              <tr
+                v-for="{ label, value } of getProperties(filter)"
+                :key="key + label"
+              >
+                <th>{{ label }}</th>
+                <td>{{ value }}</td>
+              </tr>
+            </tbody>
+          </template>
+        </v-simple-table>
+        <b v-else>Keine Attribute vorhanden!</b>
+      </v-tab-item>
+    </v-tabs-items>
+    <div class="mt-4 d-flex">
+      <v-btn
+        style="flex: 1"
+        :disabled="featureIndex <= 0"
+        :aria-disabled="featureIndex <= 0"
+        aria-label="Vorheriges Feature aufrufen"
+        @click="navigate(-1)"
+      >
+        <v-icon small>fa-chevron-left</v-icon>
+      </v-btn>
+      <v-btn
+        style="flex: 1"
+        :disabled="featureIndex >= featureItems.length - 1 || featureIndex < 0"
+        :aria-disabled="
+          featureIndex >= featureItems.length - 1 || featureIndex < 0
+        "
+        aria-label="Nächstes Feature aufrufen"
+        @click="navigate(1)"
+      >
+        <v-icon small>fa-chevron-right</v-icon>
+      </v-btn>
+    </div>
+  </v-card>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
+import { mapGetters, mapMutations, mapActions } from 'vuex'
 
 export default Vue.extend({
   name: 'DiplanGfiContent',
+  data: () => ({
+    tab: 0,
+    tables: {
+      Objektattribute: (key) =>
+        ![
+          'position',
+          'texte',
+          'refTextInhalt',
+          'externeReferenz',
+          'xpPlanName',
+          'planArtWert',
+          'xpPlanType',
+          'xpVersion',
+          'xplanMgrPlanId',
+          'wmsSortDate',
+          'begruendungsTexte',
+          'bereich',
+          'gehoertZuPlan',
+          'gehoertZuBereich',
+          'dientZurDarstellungVon',
+          'wirdDargestelltDurch',
+          'gehoertZuBP_Bereich',
+          'gehoertZuFP_Bereich',
+          'spezExterneReferenz',
+          'refScan',
+          'planinhalt',
+          'praesentationsobjekt',
+        ].includes(key),
+      Textinhalte: (key) =>
+        ['texte', 'refTextInhalt', 'begruendungsTexte'].includes(key),
+      Referenzen: (key) =>
+        [
+          'externeReferenz',
+          'spezExterneReferenz',
+          'bereich',
+          'gehoertZuPlan',
+          'gehoertZuBereich',
+          'dientZurDarstellungVon',
+          'wirdDargestelltDurch',
+          'gehoertZuBP_Bereich',
+          'gehoertZuFP_Bereich',
+          'refScan',
+          'planinhalt',
+          'praesentationsobjekt',
+        ].includes(key),
+      Planattribute: (key) =>
+        ['xpPlanName', 'xpPlanType', 'xpVersion', 'wmsSortDate'].includes(key),
+    },
+    feature: null,
+  }),
+  computed: {
+    ...mapGetters('plugin/gfi', [
+      'imageLoaded',
+      'visibleWindowFeatureIndex',
+      'windowFeatures',
+      'featureInformation',
+      'gfiConfiguration',
+    ]),
```

Comment:
```suggestion
    ...mapGetters('plugin/gfi', ['featureInformation']),
```
Values weren't used.